syft/formats/spdxjson/encoder.go

```diff
@@ -18,3 +21,20 @@ func encoder(output io.Writer, s sbom.SBOM) error {
 
 	return enc.Encode(doc)
 }
+
+func encoder2_2(output io.Writer, s sbom.SBOM) error {
+	doc := spdxhelpers.ToFormatModel(s)
+
+	var out v2_2.Document
+	err := convert.Document(doc, &out)
+	if err != nil {
+		return err
+	}
+
+	enc := json.NewEncoder(output)
+	// prevent > and < from being escaped in the payload
+	enc.SetEscapeHTML(false)
+	enc.SetIndent("", " ")
```

Comment:
more of a side note for later, not necessarily for this PR, but we should start collecting instances of these and have a common utility function somewhere in internal.

Reply:
I moved this to a common function in the `spdxjson/encoder` for now, we can always refactor more if needed.